plugins/hls-tactics-plugin/test/Utils.hs

```diff
@@ -86,7 +86,7 @@ setFeatureSet features = do
       def_config = def :: Plugin.Config
       config =
         def_config
-          { Plugin.plugins = M.fromList [("tactics",
+          { Plugin.plugins = M.fromList [("tactic",
```

Comment:
This should definitely be `"tactics"`

Reply:
Okay, so let's change descriptor to "tactics"
https://github.com/haskell/haskell-language-server/blob/2dc00d007567c2b2ce6f69f7ea9846e6fa3b6698/exe/Plugins.hs#L103